src/dotnet-roslyn-tools/Commands/CreateReleaseTagsCommand.cs

```diff
@@ -32,7 +38,9 @@ public async Task<int> InvokeAsync(InvocationContext context)
             var logger = context.SetupLogging();
             var settings = context.ParseResult.LoadSettings(logger);
 
-            return await CreateReleaseTags.CreateReleaseTags.CreateReleaseTagsAsync(settings, logger);
+            var product = context.ParseResult.GetValueForOption(ProductOption)!;
+
+            return await CreateReleaseTags.CreateReleaseTags.CreateReleaseTagsAsync(product, settings, logger);
```

Comment:
Needs more CreateReleaseTags in this line.

Reply:
Hey, I just modifed this one... didn't you originally write this? 😛

Reply:
I don't think so, and I'm not just saying that to get out of the blame!